obspy/clients/filesystem/sds.py

```diff
@@ -108,7 +113,7 @@ def __init__(self, sds_root, sds_type="D", format="MSEED",
         self.fileborder_samples = fileborder_samples
 
     def get_waveforms(self, network, station, location, channel, starttime,
-                      endtime, merge=-1, sds_type=None):
+                      endtime, merge=-1, sds_type=None, **kwargs):
```

Comment:
I added a short note in the parameter description below, which should be enough..